File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Banks/IBankData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Banks
{
    public interface IBankData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IBankPhone, IBankEMail, IBankFax
    {
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Banks/IBankData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Banks;
use Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IBankData extends IBankPhone,IBankFax,IBankEmail,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{


}